pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -31,17 +31,36 @@ local zotero = require('zotero')
 -- -- global state -- --
 local config = {
   client = 'zotero',
-  scannable_cite = false
+  scannable_cite = false,
+  csl_style=''
 }
 
+-- -- -- bibliography marker generator -- -- --
+local function zotero_bibl()
+  if (not string.match(FORMAT, 'odt')) or config.csl_style == '' then return '' end
+
+  local message = '<Bibliography: Do Zotero Refresh>'
```

Comment:
This is still in the Zotero doc prefs, right? I prefer to have that here so people don't have to guess what "Zotero Refresh" means.